x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/rules_table/utils/filters.ts

```diff
@@ -5,26 +5,26 @@
  * 2.0.
  */
 
-import { StatusFilter } from '../../../../common/types';
+import { StatusFilterBase } from '../../../../common/types';
 import type { RuleMigrationFilters } from '../../../../../../common/siem_migrations/rules/types';
-import type { FilterOptions } from '../../../types';
-import { AuthorFilter } from '../../../types';
+import type { RulesFilterOptions, RulesStatusFilter } from '../../../types';
+import { AuthorFilter, RulesSpecificStatusFilter } from '../../../types';
 
 const AUTHOR_FILTERS: Record<AuthorFilter, RuleMigrationFilters> = {
   [AuthorFilter.ELASTIC]: { prebuilt: true },
   [AuthorFilter.CUSTOM]: { prebuilt: false },
 };
 
-const STATUS_FILTERS: Record<StatusFilter, RuleMigrationFilters> = {
-  [StatusFilter.FAILED]: { failed: true },
-  [StatusFilter.INSTALLED]: { installed: true },
-  [StatusFilter.TRANSLATED]: { installed: false, fullyTranslated: true },
-  [StatusFilter.PARTIALLY_TRANSLATED]: { partiallyTranslated: true },
-  [StatusFilter.UNTRANSLATABLE]: { untranslatable: true },
-  [StatusFilter.INDEX_PATTERN_MISSING]: { missingIndex: true },
+const STATUS_FILTERS: Record<RulesStatusFilter, RuleMigrationFilters> = {
```

Comment:
Same as in dashboards.

Reply:
Yes, part of this can be a common code. I just don't want to add too much changes in one PR. I will think of how to move this part into a common in one of the following PRs.